build.gradle

```diff
@@ -18,11 +15,9 @@ apply plugin: 'org.flywaydb.flyway'
 
 jar {
     baseName = "Identity-service"
-    version = "0.1.0"
```

Comment:
why did you eliminate this?  Just move it out of the "jar" task and explicitly define an ```archiveName``` property here if you need a consistent JAR filename for the Docker build script